xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -218,8 +218,8 @@ After NuGet package v4.3.2, the default value of <xref:System.Net.DecompressionM
         <ReturnType>System.Security.Cryptography.X509Certificates.X509CertificateCollection</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the maximum allowed length of the response headers.</summary>
-        <value>The length, in kilobytes (1024 bytes), of the response headers.</value>
+        <summary>Gets the certificate collection used when authenticating using client certificates.</summary>
+        <value>A collection of X.509 certificates to use when presenting client certificates to the server.</value>
```

Comment:
See above about working with "ing".  We also usually add the type reference as a link.  How about something like "The \<see cref="T:System.Security.Cryptography.X509Certificates.X509CertificateCollection" /\>  that contains the security certificates associated with this request."?  Again, great catch!